### DISCUSSION TITLE
Is it possible to open all YouTube links via LibreTube instead of default YT app?

### DISCUSSION THREAD

**Discussion by yurizgohan** [libre-tube/LibreTube#2351] [Q&A]
As the title suggest, I want to open Vanilla YouTube links to LibreTube.
I know how to do it but I have a problem! When I put the vanilla YouTube open links setting to don't open and LibreTube one to open link it still open to vanilla Youtube.

Anyone know why ? I got some pics if you want.. (in french tho).

![Screenshot_2022-12-11-19-14-44-216-edit_com.android.settings.jpg](https://user-images.githubusercontent.com/105591137/206921250-abf00328-1425-47d7-bbcc-5db3265f62f1.jpg)

![Screenshot_2022-12-11-19-15-33-995-edit_com.android.settings.jpg](https://user-images.githubusercontent.com/105591137/206921258-4c4732d1-cbcd-4243-a514-8979e23dda01.jpg)

**Comment by enderprism** [comment]
Hi. I'm french too btw. In that menu you missed (or it maybe is only present for me) some of the ckeckboxes. These are youtube.com, m.youtube.com and www.youtube.com.
You can enable YouTube Music too :)
![Screenshot_20221215_232250_Settings.png](https://user-images.githubusercontent.com/76617157/207980681-2585116a-cc87-401e-ae54-22853413bce3.png)

**Reply by yurizgohan** [reply]
Salut ! J'ai trouvé la solution au problème il y à quelques mois / semaines, l'option que tu montres était un paramètre caché de Xiaomi. J'ai donc accéder à celle ci grâce à une application. C'est quand même très aimable, merci ! 👍🙏

EN :
Hi! I found the solution to the problem a few months/weeks ago, the option you show was a hidden Xiaomi setting. So I accessed this one through an app. That's still very kind of you, thank you! 👍🙏

**Reply by enderprism** [reply]
Glad to help :)

**Reply by AmongCrysis** [reply]
> Salut ! J'ai trouvé la solution au problème il y à quelques mois / semaines, l'option que tu montres était un paramètre caché de Xiaomi. J'ai donc accéder à celle ci grâce à une application. C'est quand même très aimable, merci ! 👍🙏
> 
> EN :
> Hi! I found the solution to the problem a few months/weeks ago, the option you show was a hidden Xiaomi setting. So I accessed this one through an app. That's still very kind of you, thank you! 👍🙏

Hi! Can you tell me which app you used to access the settings?

**Comment by AndiLeni** [comment]
I have a similar issue. On my LG device, I have set libretube to open all links in libretube. 
But this option is not shown in Firefox it chrome (missing the "open in app" button).

Any ideas what might be wrong?

**Comment by nNoidea** [comment]
While the domain is recognized, LibreTube is unable to open 'search query' links such as: [https://m.youtube.com/results?search_query=John+Wick:+Chapter+4](https://m.youtube.com/results?search_query=John+Wick:+Chapter+4) Or at least I couldn't force it to open the links. Simple video and channel links work fine tho.

**Comment by itsmmt305** [comment]
its an android 12+ "feature". xiaomi further messed it up. use [linksheet](https://apt.izzysoft.de/fdroid/index/apk/fe.linksheet) set as default web browser, open some yt link and choose libretube as - always.
